ui/app/sample-data/demoDashboard.ts

```diff
@@ -159,47 +159,19 @@ const demoDashboard: DashboardResource = {
                       kind: 'PrometheusTimeSeriesQuery',
                       spec: {
                         query:
-                          'node_memory_MemTotal_bytes{job="node",instance=~"$instance"} - node_memory_MemFree_bytes{job="node",instance=~"$instance"} - node_memory_Buffers_bytes{job="node",instance=~"$instance"} - node_memory_Cached_bytes{job="node",instance=~"$instance"}',
-                      },
-                    },
-                  },
-                },
-                {
-                  kind: 'TimeSeriesQuery',
-                  spec: {
-                    plugin: {
-                      kind: 'PrometheusTimeSeriesQuery',
-                      spec: {
-                        query: 'node_memory_Buffers_bytes{job="node",instance=~"$instance"}',
-                      },
-                    },
-                  },
-                },
-                {
-                  kind: 'TimeSeriesQuery',
-                  spec: {
-                    plugin: {
-                      kind: 'PrometheusTimeSeriesQuery',
-                      spec: {
-                        query: 'node_memory_Cached_bytes{job="node",instance=~"$instance"}',
-                      },
-                    },
-                  },
-                },
-                {
-                  kind: 'TimeSeriesQuery',
-                  spec: {
-                    plugin: {
-                      kind: 'PrometheusTimeSeriesQuery',
-                      spec: {
-                        query: 'node_memory_MemFree_bytes{job="node",instance=~"$instance"}',
+                          '1 - node_filesystem_free_bytes{job="node",instance=~"$instance",fstype!="rootfs",mountpoint!~"/(run|var).*",mountpoint!=""} / node_filesystem_size_bytes{job="node",instance=~"$instance"}',
+                        legend_format: '{{instance}} {{job}}', // figure this syntax out with Shan / Luke
```

Comment:
Wonder if just using `format` would be better or is that too generic? The thought is that this option will also be used to format the focused series names in the tooltip

Reply:
I was even thinking just `legend`, for me `legend` indicates both the text in the box that appears outside the chart and the text that appears in the tooltip... 

And we could change below to `legend_options`? 

Or another idea, `legend_name`?

I'm mostly brainstorming...

I think `format` by itself wouldn't mean a lot to me, but I get why you're trying to think about something generic

Reply:
(I'd also be fine with `legend` and `legend`, i.e. `legend` field appears inside the query schema, and inside the chart schema (and has different meanings))

Reply:
`legend_format` makes sense to me. It's really formatting the name of the individual times series so something like `series_name_format` would also make sense to me. Maybe thats better since Legend seems to be more coupled with chart visualizations? This should work for Tables as well which doesn't really include legends.

Reply:
yeah good point about tables, it's a little verbose, but I do like `series_name_format`, it makes sense for all the use-cases we've discussed so far: legends, tooltips, and tables

Reply:
I’ve started to prefer `series_name_format` since as it’s a property on the query, it will be to used to format more than just legends (see Shan’s point above), which also addresses one of Luke’s questions as well:

“If this is an option in the query, what does this option mean in the context of things that don't have a legend? Feels weird to have this on the query but have it be used in something that doesn't even have a legend?”

Reply:
Does the legend property on the TimeSeriesChart also let you format the legend? or is that only for configuring other legend properties?

Reply:
Only for configuring other legend properties initially, but I have some ideas that I want to prototype soon which would allow for more control and could require additional options.